bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -1,321 +1,1700 @@
+                recurrence: {
+                    pattern: {
+                        type: actionParameters.recurrencetype,
+                        interval: 1,
+                        daysOfWeek: [actionParameters.days],
+                        dayOfMonth: actionParameters.dayOfMonth
+                    },
+                    range: {
+                        type: "endDate",
+                        startDate: actionParameters.startDate,
+                        endDate: actionParameters.recurEndDate
+                    }
+                }
+            })
+        })
+    //console.log(JSON.stringify(responseOfcreateRecurringEventCustomTimeZone));
+    if (!responseOfcreateRecurringEventCustomTimeZone.ok) {
+        throw new Error(`createRecurringEventCustomTimeZone (${responseOfcreateRecurringEventCustomTimeZone.status}: ${responseOfcreateRecurringEventCustomTimeZone.statusText})`);
+    }
+    const user_id = [actionParameters.userId];
+    //console.log(user_id);
+    await calendarView(client, dataStore, user_id, startDate);
+    await myEvents(client, dataStore, user_id)
+};
+
+//service action for Create Recurring Office Hours with Current Timezone
+async function createRecurringOfficeHoursWithCurrentTimezone({
+    dataStore,
+    client,
+    actionParameters,
+}) {
+    const startDate = moment.utc().subtract(30, 'm').format();
+    const responseOfcreateRecurringOfficeHoursWithCurrentTimezone = await client.fetch(
+        `/v1.0/me/events`,
+        {
+            method: "POST",
+            body: JSON.stringify({
+
+                subject: actionParameters.subject,
+                body: {
+                    contentType: "HTML",
+                    content: actionParameters.content
+                },
+                start: {
+                    dateTime: actionParameters.startDateTime,
+                    timeZone: "UTC"
+                },
+                end: {
+                    dateTime: actionParameters.endDateTime,
+                    timeZone: "UTC"
+                },
+                allowNewTimeProposals: actionParameters.allowNewTimeProposals,
+                isOnlineMeeting: actionParameters.isOnlineMeeting,
+                onlineMeetingProvider: "teamsForBusiness",
+                recurrence: {
+                    pattern: {
+                        type: actionParameters.recurrencetype,
+                        interval: 1,
+                        daysOfWeek: [actionParameters.days],
+                        dayOfMonth: actionParameters.dayOfMonth
+                    },
+                    range: {
+                        type: "endDate",
+                        startDate: moment(actionParameters.startDate).format('YYYY-MM-DD'),
+                        endDate: actionParameters.enddate
+                    }
+                }
+            }
+            )
+        })
+    //console.log(JSON.stringify(responseOfcreateRecurringOfficeHoursWithCurrentTimezone));
```

Comment:
Is this `console log` really needed? Make sure you clear all the unwanted `console.logs` before you raise the PR.